lib/hammer_cli_katello/host_collection_erratum.rb

```diff
@@ -15,6 +15,12 @@ class InstallCommand < HammerCLIKatello::HostCollection::InstallContentBaseComma
         :format => HammerCLI::Options::Normalizers::List.new,
         :attribute_name => :content)
 
+      def execute
+        warn "This command uses katello agent and will be removed in favor of remote execution in a future release"
+        warn "The new command is `hammer job-invocation create --feature katello_errata_install`. Specify the host collection with the --search-query parameter"
```

Comment:
I don't remember where we left off this discussion, but I'm still in favor of renaming `job-invocation create` something more user friendly like `hammer rex`.  As it is, this messaging doesn't make clear that `hammer job-invocation create` _is_ using remote execution.  So maybe it's good to make that a bit more clear:
```suggestion
        warn "To perform the equivalent action using remote execution, use `hammer job-invocation create --feature katello_errata_install`. Specify the host collection with the --search-query parameter."
```

Reply:
I think it's implied with the first part of the warning message `... removed in favor of remove execution`

Reply:
What about `The remote execution equivalent is <command name>` ?

Reply:
I'd be fine with that!